hw/system/occamy/fpga/occamy_vcu128_bd.tcl

```diff
@@ -0,0 +1,443 @@
+# Copyright 2020 ETH Zurich and University of Bologna.
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+#
+# Nils Wistoff <nwistoff@iis.ee.ethz.ch>
+#
+# Block design generation script generated by Vivado.
```

Comment:
Is the license header ok for a script generated by a commercial tool?

Reply:
Absolutely, that is perfect :-)

Btw, how do you generate the BD? Should we maybe add this to a README or something in the `docs` folder? Like how would I change the BD in the GUI and re-generate the `tcl` script? I guess the license header should also be added manually? If that is a problem we can generate an exception in the license checker for this automatically generated file :-)

Reply:
With `occamy_vcu128.xpr` open in the Vivado GUI, the block design can be accessed on the left (`Open Block Design`). There, the block design can be viewed and modified. The `tcl` script for the block design can then by generated via `File`->`Export`->`Export Block Design`. There are some manual adjustments to this file, perhaps I should try to move these to the main script (`occamy_vcu128.tcl`). Usually I `diff` the exported with the original version of `occamy_vcu128_bd.tcl`.

I'll gladly put this info into the `docs` folder  - not very sure where, though.